api/core/v1alpha1/clustercatalog_types.go

```diff
@@ -147,17 +147,20 @@ type ClusterCatalogStatus struct {
 	//  type: Image
 	// +optional
 	ResolvedSource *ResolvedCatalogSource `json:"resolvedSource,omitempty"`
-	// contentURL is a cluster-internal URL from which on-cluster components
-	// can read the content of a catalog
+	// catalogURLs contains the URLs that can be used to access the catalog.
 	// +optional
-	ContentURL string `json:"contentURL,omitempty"`
-
+	URLs *CatalogURLs `json:"urls,omitempty"`
```

Comment:
I'd rename the type to be prefixed with the parent types name. That way we leave ourselves room in the future to have a `CatalogURLs` if we later introduce a `Catalog` type.
```suggestion
	URLs *ClusterCatalogURLs `json:"urls,omitempty"`
```